nb/ide.branding/licenseinfo.xml

```diff
@@ -33,6 +32,13 @@
         <file>core.startup/src/org/netbeans/core/startup/about_dark_nb.png</file>
         <file>core.startup/src/org/netbeans/core/startup/frame48_dark_nb.gif</file>
         <file>core.startup/src/org/netbeans/core/startup/frame_nb.gif</file>
+        <file>release/apache-netbeans.ico</file>
+        <file>release/apache-netbeans.svg</file>
+        <file>release/apache-netbeans1024.png</file>
+        <file>release/apache-netbeans256.png</file>
+        <file>release/apache-netbeans48.png</file>
+        <file>release/apache-netbeans512.png</file>
+        <file>release/netbeans.png</file>
         <file>release/netbeans.icns</file>
         <file>release/shortcuts.pdf</file>
         <file>release/shortcuts_mac.pdf</file>
```

Comment:
![image](https://github.com/user-attachments/assets/1009408e-c06e-4ad3-b2d5-ee328202b480)

the reason `netbeans.png` and `netbeans.icns` were not renamed to `apache-netbeans.png` and `apache-netbeans_mac.icns` is to avoid problems with paths downstream?

If yes: we should still consider renaming just the mac variant since it was only there for two releases and it is not like the others style wise.

Reply:
Yes, I chose to use the name apache-netbeans to be clear about the official icons, and left the two legacy paths the same, in case of external use.

And -1 to changing the semantics of the mac variant path. This has always been the path to the macOS dock icon, and added to set -Xdock:icon in the launcher script. It should remain as such. This path is certainly not two releases old! If anyone really wants another .icns file, they can add it as apache-netbeans.icns after this.

Reply:
yes the mac-os theme specific logo is 2 releases old, not the path itself which used to be the standard netbeans logo in the icns format.

netbeans_21/nb:
![image](https://github.com/user-attachments/assets/1378d5ba-bd08-4514-ad38-4cd35377361f)

why can't this be kept as in NB 21 and the mac-os theme specific icon would be added as `apache-netbeans_mac.icns`? (_mac since the shortcuts pdf already uses the postfox)

Reply:
> In case any macOS packaging (macports, homebrew, etc) still references the legacy path to get the macOS dock icon.

and [flatpak](https://flathub.org/apps/org.apache.netbeans), gnome start menus etc (https://github.com/apache/netbeans/issues/7734), since NB 22 when the change happened

i guess we could file issues against the downstream packages - but i keep wondering if simply reverting the file which was only mac specific for NB 22 and 23 and adding a dedicated mac file which is named accordingly for NB 24+ would be the easiest solution.

Reply:
That _path_ has always been mac specific from NetBeans' perspective, so yes we can provide a suitable svg/png replacement and then report issues in non-mac packages if we need to.